src/window-provider.ts

```diff
@@ -74,6 +74,9 @@ if (!window.walletRouter) {
           }
         )
       },
+      setSelectedProvider(provider: WalletProvider) {
+        this.currentProvider = provider
```

Comment:
We don't need to set it. A simple return also works. It just needs the function to be implemented. The why is a mystery though.

```suggestion
        return
```